### ISSUE TITLE
Updating screen titles to be unique

### ISSUE THREAD

**Issue by bischoffa** [department-of-veterans-affairs/va-mobile-app#9108]
**Description:** 
The [Mobile Content Accessibility Guidelines (MCAG)](https://getevinced.github.io/mcag/ ) recommend that each screen title in a mobile app should be unique. Although this is considered a minor issue in terms of impact, we should complete an audit of the app's screen titles and determine if any of the screen titles should be updated to be unique.

**References:**
- WCAG Guideline [2.4.2 Page Titled (A)](https://www.w3.org/TR/WCAG22/#page-titled)
- [Apple Human Interface](https://developer.apple.com/design/human-interface-guidelines/accessibility#Content-descriptions)
- [Android Accessibility Principles](https://developer.android.com/guide/topics/ui/accessibility/principles#a11y-pane-titles)

**AC**
- [ ] Complete an audit of the app's screen titles
- [ ] Make recommendations of where a title could and/or should be updated to be unique

**Comment by SKMOstudio** [comment]
I have completed an audit of the screen titles as read via iOS screen reader. The documented results are [here](https://docs.google.com/spreadsheets/d/1W0AvRHsNdmD624m9Ann_G2Dlxn1kQezY1gov_rxt0pU/edit?gid=1767810203#gid=1767810203). The major behaviors I saw where:

1. The title for each page was read as matching the text of the title. Since each page title is unique, it serves its purpose to differentiate the pages a user is navigating between.

2. For many pages, I observed that when a user navigated to a new page, the screen reader was reading the closes element on that page to the button they selected on the previous page to get there. This is the same behavior as documented in #10778.

3. For pages that did _not_ produce the behavior described above, when the user navigated to then, the first focused item would be the _back_ button at the top of the page. The screen reader would only read the title of the page if the user manually focused on the header.

**Comment by timwright12** [comment]
Ok, going to close this out then. Thanks!
